src/admin_sockets.rs

```diff
@@ -14,28 +14,31 @@
 
 #![cfg(target_os = "linux")]
 
-use std::str;
-use std::io::{Read, Write, Cursor};
-use std::os::unix::net::UnixStream;
-use std::net::Shutdown;
+use byteorder::{BigEndian, ReadBytesExt};
 
 use error::{RadosError, RadosResult};
-use byteorder::{BigEndian, ReadBytesExt};
+use std::io::{Cursor, Read, Write};
+use std::net::Shutdown;
+use std::os::unix::net::UnixStream;
+use std::str;
 
-/// This is a helper function that builds a raw command from the actual command. You just pass
+/// This is a helper function that builds a raw command from the actual
+/// command. You just pass
 /// in a command like "help". The returned `String` will be a JSON String.
 pub fn admin_socket_command(cmd: &str, socket: &str) -> RadosResult<String> {
-    let raw_cmd = format!("{{\"{}\": \"{}\"}}", "prefix", cmd);
-    admin_socket_raw_command(&raw_cmd, socket)
+    let raw_cmd = json!({
+        "prefix": cmd,
+    });
+    admin_socket_raw_command(&raw_cmd.to_string(), socket)
 }
 
-/// This function supports a raw command in the format of something like: `{"prefix": "help"}`.
+/// This function supports a raw command in the format of something like:
+/// `{"prefix": "help"}`.
 /// The returned `String` will be a JSON String.
 #[allow(unused_variables)]
 pub fn admin_socket_raw_command(cmd: &str, socket: &str) -> RadosResult<String> {
```

Comment:
If this is returning a JSON string, and you have Serde, would it be better to return a serde::Value instead of a string?